File: old_recipe/app/src/main/java/work/shion/androidrecipe/entities/ToDoEntity.kt
package work.shion.androidrecipe.entities

import java.util.*

/**
 * ToDoデータ
 * @param id 識別番号
 * @param isFinish 完了したかどうか
 * @param title ToDo名
 */
data class ToDoEntity(
        val id: String?,
        val isFinish: Boolean,
        val title: String,
        val updateDate: Date
)

File: WebServer/Controllers/ToDoTasks/ToDoEntity.cs
namespace WebServer.Controllers.ToDoTasks
{
    public class ToDoEntity
    {
        /// <summary>
        /// 識別番号
        /// </summary>
        public string id { get; set; }

        /// <summary>
        /// 完了フラグ
        /// </summary>
        public bool is_finish { get; set; }

        /// <summary>
        /// ToDo タイトル
        /// </summary>
        public string title { get; set; }

        /// <summary>
        /// 更新日時
        /// </summary>
        public DateTime update_time { get; set; }
    }
}
